<?xml version="1.0"?>
<launch>
    <arg name="master_name"  default="$(env ROBOT_MASTER)"/>
    <arg name="robot_name"   default="$(env ROBOT_HOST)"/>
    <arg name="target_robot" default="robot_2"/>

    <node ns="$(arg robot_name)" name="costmap_publisher" pkg="jetauto_multi" type="costmap_publish.py" required="true" output="screen">
        <param name="base_frame" value="$(arg target_robot)/base_footprint"/>
        <param name="map_frame"  value="$(arg master_name)/map"/>
    </node>
</launch>
